react-components/src/architecture/base/concreteCommands/pointCloud/PointCloudFilterCommand.test.ts

```diff
@@ -0,0 +1,109 @@
+import { assert, describe, expect, test, beforeEach } from 'vitest';
+import { PointCloudFilterCommand } from './PointCloudFilterCommand';
+import { isOdd } from '../../utilities/extensions/mathUtils';
+import { createFullRenderTargetMock } from '../../../../../tests/tests-utilities/fixtures/createFullRenderTargetMock';
```

Comment:
should be imported as

```suggestion
import { createFullRenderTargetMock } from '#test-utils/fixtures/createFullRenderTargetMock';
```